mcp/README.md

```
@@ -104,6 +104,31 @@ prompt-injection payload with a financial outcome. The plan is reported by
 
 There is also no thinking/reasoning setter yet — see "Work owned by others".
 
+### Pictures
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)